views/tables/ParticipantsTable/index.jsx

```diff
@@ -43,7 +43,7 @@ const ParticipantsTable = (props) => {
     {
       dataProperty: 'days',
       label: 'Days In Study',
-      sortable: !!sortable,
+      sortable: false,
```

Comment:
Was this sortable originally? If so I think we can support it with the Mongo $dateDiff operator: https://www.mongodb.com/docs/manual/reference/operator/aggregation/dateDiff/. If not no need to add it.

Reply:
[0af550e](https://github.com/AMP-SCZ/dpdash/pull/600/commits/0af550ebd947fb7c731cba9479f9382fa5ea9138)